src/SixLabors.Fonts/Glyph.cs

```diff
@@ -12,31 +12,28 @@ namespace SixLabors.Fonts
     {
         private readonly float pointSize;
 
-        internal Glyph(GlyphInstance instance, float pointSize)
+        internal Glyph(GlyphMetrics instance, float pointSize)
```

Comment:
minor but `metrics` is probably a clearer name than `instance`

Reply:
Yep, missed that!